API/Osu/AutomationChecks/GameAutomationChecks.cs

```diff
@@ -221,11 +170,11 @@ public static bool PassesRulesetCheck(Game game)
         }
 
         s_logger.Information(
-            "{Prefix} Tournament {TournamentId} has a game mode that differs from game, can't verify game {GameId} [Tournament: Mode={TMode} | Game: Mode={GMode}",
+            "{Prefix} Tournament {TournamentId} has a game mode that differs from game, can't verify game {GameId} [Tournament: Mode={TMode} | Game: Mode={GMode}]",
             LogPrefix,
             tournament.Id,
             game.GameId,
-            tournament.Mode,
+            (Ruleset)tournament.Mode,
             game.Ruleset
```

Comment:
```suggestion
            game.Ruleset.ToString()
```
Optional, think logs would look nicer but it's w/e